client/my-sites/plugins/plugin-settings-link/index.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+
+export default React.createClass( {
+	displayName: 'PluginSettingsLink',
+
+	propTypes: {
+		link: React.PropTypes.string.isRequired,
+	},
+
+	render() {
+		if ( ! this.props.link ) {
```

Comment:
I think this part is intentional. Since I don't ever want to show something if the user doesn't provide us with a link.